src/settlement/worker.rs

```diff
@@ -180,8 +181,26 @@ impl Settler {
                     if let Some(proof_bytes) = db.get(next_proof_key.as_bytes()) {
                         let proof_data: ProofResult = serde_json::from_slice(&proof_bytes).unwrap();
                         // verify the proof
-                        // TODO: update the new_local_exit_root
-                        let zeth_last_rollup_exit_root = settlement_provider.get_zeth_last_rollup_exit_root().await.map_err(|e| anyhow!("failed to get zeth last rollup exit root, err: {:?}", e))?;
+
+                        let bridge_service_client = Client::new();
```

Comment:
We can create a client in line 36, hence we don't have to construct it in each loop.

Reply:
We can move the get-root into a single function, then test it.

Reply:
if we can not get the response, we should retry. So it's better to store the `get-root` progress in database, and try to fetch the proof to latest as far as we can.